errai-bus/pom.xml

```diff
@@ -152,7 +152,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
-      <scope>test</scope>
+      <scope>provided</scope>
```

Comment:
`*-export-apt` modules use slf4j loggers, so I changed that to avoid the error of not having an implementation available during the execution of the annotation processors. I'm not sure that's the best solution because we'd have to change it in every module that has exportable annotations..

Reply:
Okay. Could you please add a short comment for this and the other instances? Something like this:

    <!-- Needs provided scope for use in annotation processors -->